tests/test/test_checkout_volunteer.py

```
@@ -0,0 +1,20 @@
+import pytest
+from pages.checkout_page import CheckOutPage
+from utilities.fixtures import login_with_volunteer
```

Ruff (0.12.2)

`utilities.fixtures.login_with_volunteer` imported but unused

Remove unused import: `utilities.fixtures.login_with_volunteer`

(F401)